map_org_names.yaml

```diff
@@ -1,2670 +1,3024 @@
- - ['^[[:space:]]*zauba[[:space:]]*$','Zauba Cloud']
- - ['^[[:space:]]*(zededa|zededa,?[[:space:]]inc\\.?)[[:space:]]*$','ZEDEDA, Inc.']
- - ['^[[:space:]]*zeetta[[:space:]]networks[[:space:]]*$','Zeetta Networks']
- - ['^[[:space:]]*zeit[[:space:]]*$','Zeit']
- - ['^[[:space:]]*zemoso[[:space:]]*$','ZeMoSo Technologies']
- - ['^[[:space:]]*zenefits[[:space:]]*$','Zenefits']
- - ['^[[:space:]]*zenika[[:space:]]*$','Zenika']
- - ['^[[:space:]]*zenlayer[[:space:]]*$','Zenlayer Inc']
- - ['^[[:space:]]*(zenguard|zenmate)[[:space:]]*$','ZenMate']
- - ['^[[:space:]]*zenoss[[:space:]]*$','Zenoss, Inc.']
- - ['^[[:space:]]*zensar[[:space:]]technologies[[:space:]]*$','Zensar Technologies, Inc.']
- - ['^[[:space:]]*zeomega[[:space:]]*$','ZeOmega, Inc.']
- - ['^[[:space:]]*(the[[:space:]]zephyr|zephyr)[[:space:]]*$','Zephyr Project']
- - ['^[[:space:]]*(zerolag[[:space:]]communications,?[[:space:]]inc\\.?|zerolatency)[[:space:]]*$','ZeroLag Communications, Inc.']
- - ['^[[:space:]]*zerostack[[:space:]]*$','ZeroStack']
- - ['^[[:space:]]*zewail[[:space:]]city[[:space:]]of[[:space:]]science[[:space:]]and[[:space:]]technology[[:space:]]*$','Zewail City of Science and Technology']
- - ['^[[:space:]]*zf[[:space:]]group[[:space:]]*$','ZF North America, Inc.']
- - ['^[[:space:]]*(state[[:space:]]key[[:space:]]lab[[:space:]]of[[:space:]]cad&cg|zhejiang[[:space:]]university|zju|zju-?sel)[[:space:]]*$','Zhejiang University']
- - ['^[[:space:]]*zhihu[[:space:]]*$','Zhihu']
- - ['^[[:space:]]*(zillow|zilogic)[[:space:]]*$','Zillow, Inc.']
- - ['^[[:space:]]*zimperium[[:space:]]*$','Zimperium']
- - ['^[[:space:]]*zinc[[:space:]]*$','Zinc']
- - ['^[[:space:]]*ziprecruiter[[:space:]]*$','ZipRecruiter.com']
- - ['^[[:space:]]*zix[[:space:]]*$','zixcorp.com']
- - ['^[[:space:]]*zmanda[[:space:]]*$','Zmanda']
- - ['^[[:space:]]*zoho[[:space:]]corporation[[:space:]]*$','Zoho Corporation']
- - ['^[[:space:]]*zonar[[:space:]]systems[[:space:]]*$','Zonar Systems']
- - ['^[[:space:]]*zoomdata[[:space:]]*$','Zoomdata']
- - ['^[[:space:]]*zozo[[:space:]]*$','ZOZO Technologies']
- - ['^[[:space:]]*zte[[:space:]]corporation[[:space:]]*$','ZTE Corporation']
- - ['^[[:space:]]*zulily[[:space:]]*$','Zulily, Inc.']
- - ['^[[:space:]]*zuora[[:space:]]*$','Zuora Inc.']
- - ['^[[:space:]]*zynga,?[[:space:]]inc\\.?[[:space:]]*$','Zynga, Inc.']
- - ['^[[:space:]]*école[[:space:]]de[[:space:]]technologie[[:space:]]supérieure[[:space:]]*$','École de technologie supérieure (ÉTS)']
- - ['^[[:space:]]*école[[:space:]]pour[[:space:]]*$',"École pour l'Informatique et les Techniques Av"]
- - ['^[[:space:]]*école[[:space:]]de[[:space:]]technologie[[:space:]]supérieure[[:space:]](éts)[[:space:]]*$','ÉTS - École de technologie supérieure']
- - ['^[[:space:]]*нгс[[:space:]]*$','НГС.ТЕХНОЛОГИИ (NGS.RU)']
- - ['^[[:space:]]*horizon[[:space:]]robotics[[:space:]]*$','地平线｜边缘人工智能芯片全球领导者']
+  - ['^[[:space:]]*(1&1|1&1[[:space:]]ionos[[:space:]]cloud)[[:space:]]*$','1&1 IONOS']
```

Comment:
Indenting, otherwise too big to review - I'll merge run and we will see what happens
/lgtm

Reply:
@lukaszgryglicki thanks, i just noticed the indenting and put the wip label.. lol . I have fixed that and creating another PR